tests/cluster/tests/includes/utils.tcl

```diff
@@ -7,7 +7,8 @@ proc fix_cluster {addr} {
     if {$code != 0} {
         puts $result
```

Comment:
maybe it would be a good idea to add some prefix for this print? or does it come with enough context on its own?

same for the other `puts` below.